Yosemite/Yosemite/Stores/InboxNotesStore.swift

```diff
@@ -36,8 +36,20 @@ public class InboxNotesStore: Store {
             loadAllInboxNotes(for: siteID, pageNumber: pageNumber, pageSize: pageSize, orderBy: orderBy, type: type, status: status, completion: completion)
         case .dismissInboxNote(let siteID, let noteID, let completion):
             dismissInboxNote(for: siteID, noteID: noteID, completion: completion)
-        case .dismissAllInboxNotes(let siteID, let completion):
-            dismissAllInboxNotes(siteID: siteID, completion: completion)
+        case .dismissAllInboxNotes(let siteID,
+                                   let pageNumber,
+                                   let pageSize,
+                                   let orderBy,
+                                   let type,
+                                   let status,
+                                   let completion):
```

Comment:
Nit: we can also put the `let` keyword next to `case` to omit the need of repeating `let` so many time:
```suggestion
        case let .dismissAllInboxNotes(siteID, pageNumber, pageSize, orderBy, type, status, completion):
```

Reply:
TIL! this looks much nicer, updated in https://github.com/woocommerce/woocommerce-ios/pull/6286/commits/1ba895e8ca1a0f66a62b2eac1a0e957c4e451791